app/impact-reports.server.ts

```diff
@@ -90,7 +124,7 @@ export const getHypercertClaims = async (
 			orderDirections: "asc",
 			first: 100,
 			// skip the first 2 claims (they are dummy of 0x42FbF4d890B4EFA0FB0b56a9Cc2c5EB0e07C1536 in Sepolia testnet)
-			skip: 2,
+			skip: 11,
```

Comment:
why are we skipping 11 results this time around?

Reply:
They're dummy hypercerts I created for testing. Starting at index 12, you'll see the first three actual entries from the CMS data